blocks.js

```diff
@@ -2031,7 +2032,10 @@ SyntaxElementMorph.prototype.endLayout = function () {
     %shd    - Chameleon colored rectuangular drop-down for shadowed var names
     %lst    - chameleon colored rectangular drop-down for list names
     %b      - chameleon colored hexagonal slot (for predicates)
+<<<<<<< HEAD
```

Comment:
Yeah, I accidentally commited when i was resolving the merge - they definitely shouldn't be there.

Otherwise, this can wait. It's not urgent. :)